index.html

```diff
@@ -28,13 +29,14 @@ <h2 class="site">A Minimalist Python Web Framework</h2>
             <li><a href="https://github.com/cherrypy/cherrypy">Development</a></li>
           </ul>
         </nav>
+        <a class="tidelift-link" href="https://tidelift.com/subscription/pkg/pypi-cherrypy?utm_source=pypi-cherrypy&utm_medium=referral&utm_campaign=homepage">CherryPy for Enterprise</a>
```

Comment:
I'm almost sure that this should go to <nav>. Do you have a preview to show?

Reply:
You can preview by running `python -m http.server` and accessing http://localhost:8000. Here's a screenshot I took from a rendering of this branch:

![image](https://user-images.githubusercontent.com/308610/66086954-cad46f00-e543-11e9-889c-942b8ff73303.png)

Reply:
hm.. not sure I like how it looks but I guess it's ok for now.

Reply:
Thanks. I also am not enamored of it, but it's also not horrible, and someone with more CSS-foo is welcome to iterate on the aesthetic.